### PR TITLE
Adds AOS to Overview page

### LINKED ISSUE #1124: Take out AOS on Overview page
Currently, the Overview page does not have any animations for the different sections. AOS (i.e, Animate On Scroll) effects for different sections of the Overview page would make the page much better visually.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by Akshat-Jain**
Fixes #1124 

Changes: Adds AOS effects to the different sections of the Overview page.

Demo Link: https://pr-1124-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

File: src/components/Overview/Overview.react.js [hunk 1]
```diff
@@ -19,6 +19,12 @@ import susiSkill from '../../images/susi_skill.png';
 import susiTestGif from '../../images/susi-test.gif';
 import { Link } from 'react-router-dom';
 import React, { Component } from 'react';
+import "animate.css/animate.min.css";
+import ScrollAnimation from 'react-animate-on-scroll';
+<link rel="stylesheet" href="bower_components/aos/dist/aos.css" />
```
Comment (duttaditya18) [inline src/components/Overview/Overview.react.js]: Use `import`?

File: src/components/Overview/Overview.react.js [hunk 2]
```diff
@@ -142,72 +169,75 @@ class Overview extends Component{
                    Download the App to have access to SUSI on the go.</p>
                    <div className="description__text store_icons">
                      <Link className="playstore"
-                       target="_blank" to="https://play.google.com/store/apps/details?id=org.fossasia.susi.ai"> Get It on Google Play
+                       target="_blank" to="https://play.google.com/store/apps/details?id=org.fossasia.susi.ai"> Get It on Google Play"
```
Comment (duttaditya18) [inline src/components/Overview/Overview.react.js]: What is going on with the random `"`?

Comment (Akshat-Jain) [reply src/components/Overview/Overview.react.js]: Actually this line was showing as a pink strip in sublime, so some punctuation's end part was missing, so I completed it..

File: src/components/Overview/Overview.react.js (hunk 1 above)
Comment (Akshat-Jain) [reply src/components/Overview/Overview.react.js]: Yep, I've used import... Sorry, I didn't get what you're trying to say..

File: src/components/Overview/Overview.react.js (hunk 2 above)
Comment (Akshat-Jain) [reply src/components/Overview/Overview.react.js]: Sorry, I got confused in Sublime's highlighting of this part of the code. Will remove the punctuation. Thanks for pointing it out. Sorry for the inconvenience.

File: src/components/Overview/Overview.react.js [hunk 3]
```diff
@@ -142,72 +169,75 @@ class Overview extends Component{
...
                      </Link>
                      <Link className="appstore"
-                       target="_blank" to="https://github.com/fossasia/susi_iOS"> Download on the App Store
+                       target="_blank" to="https://github.com/fossasia/susi_iOS"> Download on the App Store"
```
Comment (duttaditya18) [inline src/components/Overview/Overview.react.js]: Random `"`.

File: src/components/Overview/Overview.react.js [hunk 4]
```diff
@@ -142,72 +169,75 @@ class Overview extends Component{
...
 
+
+    	<ScrollAnimation animateIn="fadeIn" offset="300" duration="3">
         <div className="section_center">
           <div className="center__description">
             <div className="description__heading">SUSI Skills</div>
             <p className="description__text">
               SUSI is having many skills. You can look at the collection of skills at
-              <Link style={{ textDecoration: 'none' }} to="https://skills.susi.ai/" target="_blank"> skills.susi.ai</Link>.
+              <Link style={{ textDecoration: 'none' }} to="https://skills.susi.ai/" target="_blank"> skills.susi.ai</Link>."
```
Comment (duttaditya18) [inline src/components/Overview/Overview.react.js]: Random `"`.

File: src/components/Overview/Overview.react.js [hunk 5]
```diff
@@ -216,18 +246,20 @@ SUSI is intelligent to identify and answer your question in your language.
...
             <p className="description__text"><b>SUSI.AI</b> is <b>
               <Link style={{ textDecoration: 'none', color: '#000' }}
-                target="_blank" to="https://github.com/fossasia?utf8=%E2%9C%93&q=susi">Open Source</Link></b>. The code is
-                    always available for security reviews and can be improved by
-                    anyone with the knowledge and understanding online.</p>
+                target="_blank" to="https://github.com/fossasia?utf8=%E2%9C%93&q=susi">Open Source</Link></b>. The code is always available for security reviews and can be improved by anyone with the knowledge and understanding online.</p>
                     <div className="opensource-logos">
                         <span className="opensource">
-                          <Link to="https://opensource.org/" target="_blank">
+                          <Link to="https://opensource.org/" target="_blank">""
```
Comment (duttaditya18) [inline src/components/Overview/Overview.react.js]: Same here.

Comment (Akshat-Jain) [reply src/components/Overview/Overview.react.js]: Extremely sorry for the inconvenience. Fixing it right away.

File: src/components/Overview/Overview.react.js (hunk 1 above)
Comment (duttaditya18) [reply src/components/Overview/Overview.react.js]: Use `import` rather than link.

File: src/components/Overview/Overview.react.js [hunk 6]
```diff
@@ -79,11 +85,21 @@ class Overview extends Component{
...
 
     return (
+
+
+
       <div>
 
+
+
+      <script src="bower_components/aos/dist/aos.js"></script>
```
Comment (duttaditya18) [inline src/components/Overview/Overview.react.js]: Same here.

**Comment by Akshat-Jain** [comment]
@duttaditya18 @isuruAb Can you please review and merge ? :)

**Comment by duttaditya18** [comment]
@Akshat-Jain Good job on using `animate.css`. We can change the animation if needed as well!

👍

**Comment by duttaditya18** [comment]
Also merge this branch to the latest.

**Comment by Akshat-Jain** [comment]
@duttaditya18 Merged this branch to the latest. Please review and merge :)

**Comment by mariobehling** [comment]
The effect is too slow and appears on scroll down and scroll up. It should be much simpler. Right now it is distracting. Therefore I am reverting this.

**Comment by mariobehling** [comment]
Unfortunately, it cannot be reverted. @Akshat-Jain Please take out this effect and keep the design simpler. Get inspired by Google. They don't use these kind of effects.

**Pull request merged**